src/main/java/org/web3j/console/project/InteractiveOptions.java

```diff
@@ -34,13 +36,21 @@
     }
 
     protected final String getProjectName() {
-        print("Please enter the project name (Required Field): ");
-        return getUserInput();
+        print("Please enter the project name (Required Field):");
+        String projectName = getUserInput();
+        while (!InputVerifier.classNameIsValid(projectName)) {
+            projectName = getUserInput();
```

Comment:
It doesn't look like this would give the user any feedback if invalid?

If it's not valid something indicating this needs to be printed if that doesn't happen, same for the package name..

Reply:
Ok, so input verifier does print feedback to console. As discussed we can look to improve on this to make it more obvious, but at least there's some feedback given.